cli/nep_test/nep17_test.go

```diff
@@ -64,12 +65,17 @@ func TestNEP17Balance(t *testing.T) {
 		e.CheckNextLine(t, "^\\s*$")
 	})
 	t.Run("all accounts", func(t *testing.T) {
+		getUtilityTokenBalance := func(acc util.Uint160) *big.Int {
+			_, err := e.Chain.GetTokenLastUpdated(acc)
+			require.NoError(t, err)
+			return e.Chain.GetUtilityTokenBalance(acc)
+		}
```

Comment:
How this could prevent the described test failure? Have you checked, is it a race or something else? Why the race is happening (if it's a race)?

Reply:
I'm asking because I'd like to make 100% sure that we found and fixed the source of the problem correctly.

Reply:
the initial hypothesis was that if on the server side, we do GetTokenLastUpdated(acc), then we need it in test https://github.com/nspcc-dev/neo-go/blob/master/pkg/services/rpcsrv/server.go#L1185 
however, with the last changes here, I could reproduce the failure of the test.